packages/webapp/src/utils/utils.tsx

```diff
@@ -146,6 +146,27 @@ export function formatDateToUSFormat(dateString: string): string {
     return formattedDate;
 }
 
+export function formatDateToIntFormat(dateString: string): string {
```

Comment:
nitpick: 
`formatDateToIntlFormat` maybe with a `L`. At first at read it as `To Integer Format`
or maybe something like
```
const Intl = {
    formatDate: (dateString: string): String { ...}
````

Reply:
ah yeah, I knew it was bad when writing it :D